ghost/i18n/locales/ru/portal.json

```diff
@@ -1,173 +1,173 @@
+    "Here are a few other sites you may enjoy.": "Вот несколько других сайтов, которые могут вам понравиться.",
+    "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Если рассылка помечена как спам, электронные письма перестают отправляться на этот адрес и автоматически отключаются, чтобы вы больше не получали нежелательных сообщений.",
+    "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Если жалоба на спам была случайной или вы хотите снова получать рассылку, вы можете повторно подписаться на рассылку, нажав на кнопку «Подписаться повторно» на предыдущем экране.",
+    "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Если вы отмените подписку сейчас, доступ к содержимому сохранится у вас до {{periodEnd}}.",
+    "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Если вы указали корпоративный или принадлежащий государственным органам адрес электронной почты, обратитесь в свой ИТ-отдел и попросите разрешить получать электронные письма от {{senderEmail}}.",
+    "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Если вы хотите снова получать электронные письма, лучшим следующим шагом будет проверить ваш email адрес на наличие проблем, а затем нажать кнопку «Подписаться повторно» на предыдущем экране.",
+    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Если вы не получаете рассылку, на которую подписаны, вот несколько вещей, которые следует проверить.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Если вы выполнили все эти проверки, но по-прежнему не получаете электронные письма, вы можете обратиться за поддержкой по адресу: {{supportAddress}}.",
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "В случае возникновения постоянной ошибки при попытке отправки рассылки — электронные письма будут отключены для данной учётной записи и указанного email адреса.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "В вашем почтовом клиенте/приложении добавьте {{senderEmail}} в список контактов. Это скорее всего сигнализирует вашему почтовому провайдеру, что письмам, отправленным с этого адреса, следует доверять.",
+    "Invalid email address": "Неправильный email адрес",
+    "Jamie Larson": "Павел Бид",
```

Comment:
Whats the reason for the name change here?

Reply:
Same reason - same name.

Firstly: it was funny shorten form of a name and surname. Similar to: John Dick.

Secondly: According to the guideline here should be a typical unisex name. The thing is in Russian no unisex names because of lang rules. We have a few unisex names, but it is different when you write it (Alexander (male) and Alexandra (female)).